en/docs/insights/programmatic-access-choreo-insights-api.md

```diff
@@ -0,0 +1,62 @@
+# Programmatic Access to Insights API
+
+The **Insights** page in the Choreo Console fetches data from the Insights API exposed over the Internet. However, in some scenarios, you may need to fetch data from external systems and implement custom dashboards to display this data.
```

Comment:
Doesn't seem correct to say "The **Insights** page in the Choreo Console fetches data. . . ". The term page doesn't seem appropriate I feel.

Reply:
We have a separate page for insights in the console. So why it is wrong to use that term? Or else do you have any suggestion for this?

Reply:
When we have it as "The Insights page fetches data from the Insights API. . ." it is not quite right because the Insights page is only displaying the data. Not really fetching the data.
We would have to say something similar  
The Insights page in Choreo displays data fetched from the Insights API exposed over the internet.

Reply:
Fixed with [15541b0](https://github.com/wso2/docs-choreo-dev/pull/437/commits/15541b0254dadedf8213c084c17bfd6498b6053d)